HAM/data_scientist/assets/algo/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# this base image works in both CPU and GPU enabled environments
+FROM substrafoundation/substra-tools:0.7.0
+
+RUN mkdir /.cache
+RUN chmod -R 777 /.cache
```

Comment:
Simple question, I didn't test, but isn't 755 sufficient?

Reply:
I honestly don't know. However it would be worthwhile to test.

Reply:
```suggestion
RUN chmod -R 755 /.cache
```